### ISSUE TITLE
Add mapping example for dateOther without a type

### ISSUE THREAD

**Issue by arcadiafalcone** [sul-dlss-labs/cocina-descriptive-metadata#105]
<originInfo displayLabel="Acquisition date">
            <dateOther keyDate="yes" encoding="w3cdtf">1970-11-23</dateOther>
          </originInfo>

**Comment by ndushay** [comment]
from Slack conversation w Arcadia, the correct cocina json is:

```
         {
            "displayLabel": 'Acquisition date',
            "date": [
              {
                "value": '1970-11-23',
                "encoding": {
                  "code": 'w3cdtf'
                },
                "status": 'primary'
              }
            ]
          }
```

**Comment by arcadiafalcone** [comment]
Closed by #110
